simple_certmanager/models.py

```diff
@@ -71,26 +65,28 @@ def __str__(self):
     def save(self, *args, **kwargs):
         # generate private key once
         if not self.private_key:
-            self.private_key, self.csr = generate_private_key_with_csr(
+            self.private_key, csr = generate_private_key_with_csr(
```

Comment:
Or just

```suggestion
            self.private_key = generate_private_key(
```

and delete `generate_private_key_with_csr`, I don't think it's used anymore